superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigForm/ColumnSelect.tsx

```diff
@@ -103,6 +128,7 @@ export function ColumnSelect({
       onChange={onChange}
       options={options}
       placeholder={t('Select a column')}
+      notFoundContent={t('No columns found')}
```

Comment:
Should we make this message display the `currentFilterType` when set, just in case? Something like "No numerical columns found" if filtering for numerical columns etc.

Reply:
Replaced with `No compatible columns found`, adding `currentFilterType` would require an ugly name mapping